std_msgs/Header header
float64 ship_heading                 # ship_heading reading
float64 ship_heading_variance        # ship_heading_variance reading
